spec/support/vcr_cassettes/profile/demographics.yml

```diff
@@ -0,0 +1,56 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://int.vet360.va.gov/demographics/demographics/v1/2.16.840.1.113883.4.349/9d018700-b72c-444a-95b4-43e14a4509ea%5EPN%5E200VIDM%5EUSDVA
```

Comment:
Could this file live in the same folder as `spec/support/vcr_cassettes/va_profile/demographics/demographics.yml`, since they both are recordings of the API interactions with `https://int.vet360.va.gov/demographics/demographics/v1`?

Reply:
It can live anywhere, but I thought the location was based on the test and not the content of the cassette. Can you confirm?

Reply:
Location of VCR cassettes tends to be by the test that uses them. I think this location matches that pattern.